dir.props

```diff
@@ -75,7 +75,8 @@
 
   <!-- Package dependency validation -->
   <PropertyGroup>
-    <ValidatePackageVersions>true</ValidatePackageVersions>
+    <!-- Temporarily disabled the validation -->
```

Comment:
There is no good way to disable it only at an individual level. The property is not wired correctly. Since, this is only for a day, I thought this will be easiest